Source/Core/VideoBackends/D3D/D3DBase.cpp

```diff
@@ -206,8 +207,10 @@ std::vector<u32> GetAAModes(u32 adapter_index)
   for (u32 samples = 1; samples <= D3D11_MAX_MULTISAMPLE_SAMPLE_COUNT; ++samples)
   {
     UINT quality_levels = 0;
-    if (SUCCEEDED(temp_device->CheckMultisampleQualityLevels(DXGI_FORMAT_R8G8B8A8_UNORM, samples,
-                                                             &quality_levels)) &&
+    if (SUCCEEDED(temp_device->CheckMultisampleQualityLevels(
+            D3DCommon::GetDXGIFormatForAbstractFormat(FramebufferManager::GetEFBColorFormat(),
```

Comment:
Please pull this out of the for loop.  In addition, it'll make this a little more readable:

`const auto format = D3DCommon::GetDXGIFormatForAbstractFormat(FramebufferManager::GetEFBColorFormat(), false);`

then

```
for (u32 samples = 1; samples <= D3D11_MAX_MULTISAMPLE_SAMPLE_COUNT; ++samples)
  {
    UINT quality_levels = 0;
    if (SUCCEEDED(temp_device->CheckMultisampleQualityLevels(format, samples, &quality_levels)) && quality_levels > 0)
    {
    ...
```